[SYCL] Add possibility to specify DPC++ RT directory (#55)
intel/llvm-test-suite: SYCL/README.md

***CMAKE_CXX_COMPILER*** should point to the DPCPP compiler

***DPCPP_RT_LIBRARY_DIR*** (optional) if present sets location for DPC++ runtime

>  libraries to be used on the test execution. The test built is done with

Reviewer: ```suggestion
 libraries to be used during tests execution. Tests are built with
```
